--- custom_config.inc.php.example.orig	2014-12-01 11:57:08.150607542 +0000
+++ custom_config.inc.php.example	2014-12-01 11:59:38.880060033 +0000
@@ -37,6 +37,12 @@
 //
 // $tlCfg->log_path = '/var/testlink-ga-testlink-code/logs/'; /* unix example */
 // $g_repositoryPath = '/var/testlink-ga-testlink-code/upload_area/';  /* unix example */
+
+// logging path
+$tlCfg->log_path = '/var/log/testlink/';
+// file upload path
+$g_repositoryPath = '/var/www/testlink/upload_area/';
+
 //
 // 
 // ********************************************************************************************
@@ -232,4 +238,4 @@
 // $tlCfg->req_spec_template->scope->value = 'c:\usr\local\xampp-1.7.2\xampp\htdocs\head-20090909\item_templates\req_spec.txt';
 
 
-?>
\ No newline at end of file
+?>
